web/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/web/rest/dialog/DialogController.kt

```diff
@@ -92,6 +92,23 @@ class DialogController @Inject constructor(
                 }
     }
 
+    @POST
+    @Path("/sendinfomelding")
+    fun sendInfomelding(
+            @Context request: HttpServletRequest,
+            @PathParam("fnr") fnr: String,
+            infomeldingRequest: InfomeldingRequest
+    ): Response {
+        return tilgangskontroll
+                .check(Policies.tilgangTilBruker.with(fnr))
+                .get(Audit.describe(CREATE, Person.Henvendelse.Les, AuditIdentifier.FNR to fnr)) {
+                    val context = lagSendHenvendelseContext(fnr, request)
```

Comment:
method sendinfomelding , sendreferat, sendspørsmål gjøre samme ting. Kan man ikke lage et felles method ?

Reply:
De er ikke helt like, og tilhører vel hvert sitt endepunkt. Men sikkert muligheter for å gjøre metodene mer generiske :)

Reply:
Kan evt håndteres i en egen PR.